pallets/gear/src/benchmarking/mod.rs

```diff
@@ -485,13 +486,22 @@ benchmarks! {
         let block_count = 1_000u32.into();
 
         init_block::<T>(None);
-    }: _(RawOrigin::Signed(caller.clone()), program_id, block_count)
+
+        let mut result = Ok(().into());
+    }: {
+        let _ok = result.is_ok();
```

Comment:
Why can't we adjust config value for benchmark or use similar approach as in the verify section?
```suggestion
```

Reply:
discussed: will enable rent logic under cfg(runtime-benchmarks)